app/models/components/system/admin/system_admin_ability_component.rb

```diff
@@ -2,9 +2,7 @@ module System::Admin::SystemAdminAbilityComponent
   include AbilityHost::Component
 
   def define_permissions
-    if user
-      do_not_allow_system_admin_manage_default_instance
-    end
+    do_not_allow_system_admin_manage_default_instance
```

Comment:
OK, I thought the user must there, otherwise no ability is defined...might understand wrongly.